### PR TITLE
Add HW22

### PR DESCRIPTION
**Pull request by Gritskov-Vitaliy**

### CODE REVIEW

File: HW22/roles/hw22/tasks/users.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+   name: "{{ item.name }}"
+   shell: "{{ item.shell }}"
+   home: "{{ item.home_dir }}"
+   groups: "{{ item.groups }}"
+   password: "{{ user_pass }}"
+   state: present
+  loop: "{{ users }}"
+  no_log: true
+  become: true
+  when: user_add_method == "loop"
```
Comment (mavrik90) [inline HW22/roles/hw22/tasks/users.yml:20]: Nice one!

File: HW22/roles/hw22/tasks/hosts_file.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+- name: Find out host ip
+  ansible.builtin.set_fact: host_ip={{ ansible_ssh_host }}
```
Comment (mavrik90) [inline HW22/roles/hw22/tasks/hosts_file.yml:2]: Which another option you have to get host ip?

**Review by mavrik90 — APPROVED** [review]
Well done!

File: HW22/roles/hw22/tasks/hosts_file.yml (hunk 2 above)
Comment (Gritskov-Vitaliy) [reply HW22/roles/hw22/tasks/hosts_file.yml:2]: host_ip=hostvars[inventory_hostname]['ansible_env'].SSH_CONNECTION.split(' ')[2]

**Pull request merged**